.github/workflows/veracode.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
 
       #Copy the jar to directory which gets uploaded to Veracode
       - name: Copy Pool JAR
-        run: cp target/value-added-service-1.0.0.jar target/veracode/value-added-service-1.0.0.jar
+        run: cp target/value-added-service-1.0.4.jar target/veracode/value-added-service-1.0.4.jar
```

Comment:
Can't you use a wildcard here for the copy command or somehow construct the path based on variables?
Updating the build workflow each time you build a new version seems like a bad idea and can easily be forgotten

Reply:
Changed to a not hardcoded version